Sources/Sentry/UIView+Sentry.m

```diff
@@ -1,34 +0,0 @@
-#import "UIView+Sentry.h"
-
-#if SENTRY_HAS_UIKIT
```

Comment:
`m`: Are you sure we don't need this file anymore?

Reply:
see https://github.com/getsentry/sentry-cocoa/pull/3175#discussion_r1347755645